examples/visualization/example_jupyter.ipynb

```diff
@@ -124,962 +124,7 @@
    "outputs": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    sat1 = ActorBuilder.get_actor_scaffold(
</pre><p>This throws a type error, remove the position of sat1, sat2, sat3, and sat4 when getting the scaffold for the notebook to run.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
fixed